code/game/machinery/Sleeper.dm

```diff
@@ -16,6 +16,7 @@
 	var/efficiency = 1
 	var/min_health = -25
 	var/list/available_chems
+	var/mob/living/simple_animal/ambush_inside = null //whether there is a boogeyman hiding in your sleeper and what type of boogeyman is it.

```

Comment:
@KorPhaeron i mean if many mobs that do have this behaviour are gonna do this then it's worth it to have a var like that no?